pycytominer/cyto_utils/load.py

```diff
@@ -69,13 +77,66 @@ def infer_delim(file: str):
     return dialect.delimiter
 
 
-def load_profiles(profiles):
+def is_anndata(
+    path: Union[str, pathlib.Path, ad._core.anndata.AnnData],
+) -> tuple[bool, Optional[str]]:
+    """Return True if ``path`` contains an AnnData dataset (H5AD or Zarr).
```

Comment:
```suggestion
    """
```

This is redundant with below and also not accurate (it returns a tuple, described well below)